src/core/js/views/drawerView.js

```diff
@@ -86,7 +86,14 @@ define([
             this._isCustomViewVisible = true;
             Adapt.trigger('drawer:empty');
             this.showDrawer();
+
             this.$('.drawer-holder').html(view);
+
+            var model = this.collection.findWhere({
```

Comment:
@barry-learningpool it works fine for resource drawer, but  thinking forward , when i think about PLP ``` this.collection ``` always have resource object in it https://github.com/adaptlearning/adapt-contrib-resources/blob/master/js/adapt-contrib-resources.js#L29-L35 . and it never gets empty , when you click on any other drawer item,  undefined will be return